internal/encryption/targets.go

```diff
@@ -128,3 +132,36 @@ func (e *targetBuilder) build(target *config.TargetConfig, targetName string) (m
 
 	return methods, diags
 }
+
+func (base *baseEncryption) setupTargetEncCfg() (*config.EncryptionConfig, hcl.Diagnostics) {
+	var diags hcl.Diagnostics
+	evalCtx := &hcl.EvalContext{
+		Variables: map[string]cty.Value{},
+	}
+	methodAddrs, metDiags := base.extractMethodsAddrs()
+	diags = append(diags, metDiags...)
+	if diags.HasErrors() {
+		return nil, diags
+	}
+	evalCtx.Variables["method"] = cty.ObjectVal(methodAddrs)
```

Comment:
What is this doing?

Reply:
This is needed for the `gohcl.DecodeExpression` inside `requiredMethods`.